internal/boxcli/generate.go

```diff
@@ -110,7 +111,9 @@ func sshConfigCmd() *cobra.Command {
 		Long:   "Check ssh config and if they don't exist, it generates the configs necessary to connect to devbox cloud VMs.",
 		Args:   cobra.MaximumNArgs(0),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runGenerateCmd(cmd, flags)
+			// ssh-config command is exception and it should run without a config file present
+			_, err := cloud.SSHSetup(flags.githubUsername)
+			return errors.WithStack(err)
```

Comment:
@hezhizhen is this gonna work when there is no error? I mean what would be the output of this function if it returns `errors.WithStack(nil)`?

Reply:
It will return nil.

```go
// WithStack annotates err with a stack trace at the point WithStack was called.
// If err is nil, WithStack returns nil.
func WithStack(err error) error {
	if err == nil {
		return nil
	}
	return &withStack{
		err,
		callers(),
	}
}
```